test/unit/server.js

```diff
@@ -424,8 +413,28 @@ describe('Server', function () {
 
   describe('start', function () {
     describe('without tasks', function () {
-      it('should fail', function () {
-        return assert.isRejected(server.start(), /handler not defined/)
+      beforeEach(function () {
+        sinon.stub(ponosDefaultLogger, 'warn')
+      })
+
+      afterEach(function () {
+        ponosDefaultLogger.warn.restore()
+      })
+
+      it('should warn', function () {
+        server.start().then(function () {
+          sinon.assert.calledTwice(ponosDefaultLogger.warn)
+          sinon.assert.calledWith(
+            ponosDefaultLogger.warn.getCall(0),
```

Comment:
doesn't matter, just pointing it out - I don't think I use any ordering in my checks (and there are very few where I check that it was called twice with two things). This should be okay since the order it checks in will always be the same